sale_commission_pricelist/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==============================
+Sales commissions by pricelist
+==============================
+
+This module extends the functionality of sale_commission to allow you set a
+commission to pricelist item.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to pricelist
+#. Set commission
+#. Create sale order
```

Comment:
I've tested @carlosdauden and I'm not sure if it works:

- I set 20% commision for pricelist but
- Sale order takes 10% commision

![2018-02-20_15-10-20](https://user-images.githubusercontent.com/8736623/36428501-5cb69bf6-1650-11e8-9f40-e6f9e25221c8.png)

Reply:
Are you surf that you know steps to use sale_commission?
Try adding Agent to partner :wink: